tracing/vm-tracing/v8-tracing.md

```diff
@@ -0,0 +1,39 @@
+## V8/Chromium Tracing  
+Generic macro definitions intended to be shared by many projects are in `trace_event_common.h`.  
+Source of truth for `trace_event_common.h` is in `chromium:src/base/trace_event/trace_event_common.h`.  
+This file is in Chromium but not in V8. @ofrobots copied it into `nodejs:deps/v8`. Perhaps it should be moved to `nodejs:deps/v8/include` or `nodejs:deps/v8/src/trace_event`.  
+
+Implementation-specific ("INTERNAL_*") macro definitions are in `v8:src/trace_event.h` and `chromium:src/base/trace_event.h`.  
+Chromium: `trace_event.h` macros utilize `v8::base::TraceLog::AddTraceEvent` to write traces.  
+V8: `trace_event.h` macros utilize `v8::Platform::AddTraceEvent` to write traces.  
+Node uses `DefaultPlatform` from `v8:src/libplatform/default-platform.h` which currently simply returns 0.  
+A simplistic implementation which logs all trace events to stdout illustrates the basics:
+
+```cpp
```

Comment:
Looks like `cpp` is a proper alias in [the list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L415) so it's probably because the snippet is incomplete. I'll set it to plain text for now.